test/aspnetcore_schema.xml

```diff
@@ -0,0 +1,39 @@
+<!--
```

Comment:
How easy would it be to get the schema file from the Microsoft.AspNetCore.AspNetCoreModuleV2 package? This solution isn't maintainable though we most likely won't update the schema any longer.

Reply:
I think this should become part of the IIS test deployers, not KoreBuild. The deployer  could check for the updated schema, attempt to update it if it's out of date, and maybe fail the test if it doesn't have the right permissions.